spec/system/passenger_show_spec.rb

```diff
@@ -14,18 +13,16 @@
     context 'viewing passenger' do
       let(:number) { @passenger.phone.delete('-') }
       context 'passenger is subscribed to sms' do
-        it 'displays a SMS address ' do
-          @passenger.update(subscribed_to_sms: true, carrier: @carrier)
+        it 'say so' do
```

Comment:
Maybe a more descriptive test name would be good here lol.

Reply:
First of all, this isn't a change, I just moved stuff. Second of all, it's supposed to be read in conjunction with the context. Third of all, I don't care, so I changed it.